README.md

```diff
@@ -1,58 +1,122 @@
-<img src="https://storage.googleapis.com/passage-docs/passage-logo-gradient.svg" alt="Passage logo" style="width:250px;"/>
+![passage-android](https://storage.googleapis.com/passage-docs/github-md-assets/passage-android.png)
 
-### Native passkey authentication for your Android app
-## Welcome!
-Integrating passkey technology can be really hard. That's why we built the Passage Android SDK - to make passkey authentication easy for you and your users, all in a native Android experience.
+[![Android](https://img.shields.io/badge/Android-3DDC84?logo=android&logoColor=white)](#) [![Kotlin](https://img.shields.io/badge/Kotlin-%237F52FF.svg?logo=kotlin&logoColor=white)](#) ![GitHub License](https://img.shields.io/github/license/passageidentity/passage-android)
+![Static Badge](https://img.shields.io/badge/Built_by_1Password-grey?logo=1password)
 
-<img width="700" height="1100" alt="image" src="https://github.com/user-attachments/assets/674fd6e6-0330-42e2-b467-eada7709f509">
+## About
 
-<br>
-<br>
+[Passage by 1Password](https://1password.com/product/passage) unlocks the passwordless future with a simpler, more secure passkey authentication experience. Passage handles the complexities of the [WebAuthn API](https://blog.1password.com/what-is-webauthn/), and allows you to implement passkeys with ease.
 
-## Usage
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
+
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
+
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
+
+### In passage-android
+
+Use passage-android to implement [passage-android/passage](https://github.com/passageidentity/passage-android/passage) in your Android application to authenticate requests and manage users, [passage-android/passageflex](https://github.com/passageidentity/passage-android/passageflex) in your existing auth to use passkeys for registration, authentication, or added security on sensitive user actions, and [passage-android/authentikit](https://github.com/passageidentity/passage-android/authentikit) to gather data on your users' passkey readiness.
```

Comment:
This is a mouthful but what I came up with that didn't sound super redundant given the above. Ppen to suggestions!